tools/partutil/helpers.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package partutil
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+)
+
+// ExecCmdToStdout runs a command and output to stdout.
+func ExecCmdToStdout(cmdLine string) error {
+	cmd := exec.Command("/bin/bash", "-c", cmdLine)
```

Comment:
Is the /bin/bash necessary? Can the cmdLine be run directly?

Reply:
Talked in the daily meeting